config/jobs/kubernetes/sig-release/kubernetes-builds.yaml

```diff
@@ -19,8 +19,11 @@ presubmits:
         securityContext:
           privileged: true
         resources:
+          limits:
+            cpu: 7300m
+            memory: "41Gi"
```

Comment:
Any reason for going higher than recommended `36Gi`?

Reply:
actually not, will fix that

Reply:
done